paddle/fluid/recordio/README.md

```diff
@@ -0,0 +1,13 @@
+## Background
+
+RecordIO is a file format as a container of records.  This package is a C++ implementation of https://github.com/paddlepaddle/recordio, which originates from https://github.com/wangkuiyi/recordio.
```

Comment:
Maybe reword to "The RecordIO file format is a container for records."

Reply:
Done